src/bin/flamegraph.rs

```diff
@@ -14,6 +14,12 @@ struct Opt {
     #[structopt(long = "nameattr")]
     nameattr_file: Option<PathBuf>,
 
+    /// The `--inverted` flag.
+    /// Changes the flame graph such that the entire plot is up-side-down.
+    /// Also, changing the title of the plot.
```

Comment:
Actually, sorry, one more thing. Let's make this a little less noisy, and make it just:
```rust
/// Plot the flame graph up-side-down.
```

Reply:
Done